deployments/ethereum.json

```diff
@@ -4761,6 +4761,471 @@
         }
       ]
     },
+    "WeETHOracle_Equivalence": {
```

Comment:
I think the three entries for `WeETHOracle_NonEquivalence` should be in this file